shop.html

```diff
@@ -1 +1,127 @@
-
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Shop</title>
+  </head>
+  <body>
+    <header>
+      <label><a href="/">Shop</a></label>
+      <input type="search" />
+      <button>Search</button>
+    </header>
+    <hr />
+    <nav>
+      <a href="#Photo_video">Photo &amp; Video</a> |
+      <a href="#Change_the_way">Change the Way You Use Light</a> |
+      <a href="#Product_information">Product information</a>
+    </nav>
+    <hr />
+    <main>
+      <h2>
+        Philips Hue White and Color Ambiance A19 60W Equivalent LED Smart Bulb
+        Starter Kit
+      </h2>
+      <label>Price: 100 &euro; </label>
+      <button>Buy now</button>
+      <ul>
+        <li>
+          <a href="buy@shop.com">Order via email buy@shop.com</a>
+        </li>
+        <li>
+          <a href="(111)1111111">Order via call (111)1111111</a>
+        </li>
+      </ul>
+      <h2 id="Photo_video">Photo &amp; Video</h2>
```

Comment:
я бы оборачивал все 3 части в `<section>`
```suggestion
      <section>
          <h2 id="Photo_video">Photo &amp; Video</h2>
          ...
```